app/scripts/store/dimStoreItems.directive.js

```diff
@@ -80,25 +79,49 @@
         '</div>'
       ].join('')
     };
-
-    function Link(scope, element, attrs) {
-      scope.vm.onDrop = function(id, $event, equip) {
-        var vm = scope.vm;
-
-        var srcElement = $('#' + id);
-
-        vm.moveDroppedItem(angular.element(srcElement[0])
-          .scope()
-          .item, equip);
-      };
-    }
   }
 
-  StoreItemsCtrl.$inject = ['$scope', 'loadingTracker', 'dimStoreService', 'dimItemService', '$q', '$timeout', 'toaster', 'dimSettingsService'];
 
-  function StoreItemsCtrl($scope, loadingTracker, dimStoreService, dimItemService, $q, $timeout, toaster, dimSettingsService) {
+  StoreItemsCtrl.$inject = ['$scope', 'loadingTracker', 'dimStoreService', 'dimItemService', '$q', '$timeout', 'toaster', 'dimSettingsService', 'ngDialog', '$rootScope'];
+
+  function StoreItemsCtrl($scope, loadingTracker, dimStoreService, dimItemService, $q, $timeout, toaster, dimSettingsService, ngDialog, $rootScope) {
     var vm = this;
 
+    // Detect when we're hovering a dragged item over a target
+    var dragTimer = null;
+    var hovering = false;
+    var dragHelp = document.getElementById('drag-help');
```

Comment:
I could do that, but it'd require communicating with a separate directive through `$rootScope` or `$broadcast`. Since this is so simple, I figured I'd just do it right where the other code is.